x-pack/plugins/cloud_security_posture/public/pages/benchmarks/benchmarks.tsx

```diff
@@ -152,8 +152,9 @@ export const Benchmarks = () => {
 
   const queryResult = useCspBenchmarkIntegrationsV2();
   const benchmarkResult =
-    queryResult.data?.items.filter((obj) => getBenchmarkCisName(obj.id)?.includes(query.name)) ||
-    [];
+    queryResult.data?.items.filter((obj) =>
+      getBenchmarkCisName(obj.id)?.toLowerCase().includes(query.name.toLowerCase())
+    ) || [];
```

Comment:
This is a good improvement, but every time filter is called, `query.name.toLowerCase()` is executed. This can be computed once and reused. It's better to compute it outside the filter function:

```js
const lowerCaseQueryName = query.name.toLowerCase();
const benchmarkResult = queryResult.data?.items.filter((obj) =>
    getBenchmarkCisName(obj.id)?.toLowerCase().includes(lowerCaseQueryName)
) || [];
```